repose-aggregator/src/docs/asciidoc/recipes/quick-start.adoc

```diff
@@ -54,7 +54,7 @@ Let's break that command down and take a closer look at what it is doing:
 <1> This is the part of the command that tells Docker to create and start a container.
 <2> This option will run the container in detached mode.
     In detached mode, the container will run in the background.
-    If we were to run in attached mode (the default) instead of detached mode, the terminal we used to execute this command would receive all of the container output (e.g. STDOUT, and STDERR).
+    If we were to run in the default attached mode instead of optional detached mode, the terminal we used to execute this command would receive all of the container output (e.g. STDOUT, and STDERR).
```

Comment:
I would also drop the optional here.

Really, I wonder if there's a way to include this content rather than duplicating it.

Reply:
I had looked at it, but it is only one callout. If anything, I would just remove the callout from the Docker recipe since we state the assumption is that the reader has already gone through the Quick Start.